iron-selectable.html

```diff
@@ -148,7 +149,7 @@
       this._observer = this._observeItems(this);
       this._updateItems();
       if (!this._shouldUpdateSelection) {
-        this._updateSelected(this.attrForSelected,this.selected)
+        this._updateSelected(this.selected);
```

Comment:
We might as well remove the `this.selected` argument as well given that `_updateSelected` does nothing with it.